ml/base/estimator.py

```diff
@@ -52,31 +41,34 @@ def _gen_estimator(self):
         )
         return estimator
 
-    def _gen_model_fn(self):
+    def _gen_model_fn(self, gpu_id=2):
 
         def model_fn(features, labels, mode, params):
-            # Construct graph.
-            self.output = self.graph.add_forward_pass()
+            # TODO(jdaaph): Add cpu and gpu flag.
+            with tf.device('/gpu:{}'.format(gpu_id)):
```

Comment:
Is there a more elegant way of registering variables to some certain device / namespace of tf?

Reply:
I put that as an issue #28